{"body": "I'm getting an error on the server: \"The joined path <a bunch of webpage generated source code>  is located outside of the base path component (/home/mythmon/src/normandy/normandy/media)\"\n\nThe editor isn't showing up now. I am getting 404s for the font awesome css file. I tried running `manage.py collectstatic`. I'm not sure what's going on.\n", "body_text": "I'm getting an error on the server: \"The joined path   is located outside of the base path component (/home/mythmon/src/normandy/normandy/media)\"\nThe editor isn't showing up now. I am getting 404s for the font awesome css file. I tried running manage.py collectstatic. I'm not sure what's going on.", "url": "https://api.github.com/repos/mozilla/normandy/issues/comments/186398756", "created_at": "2016-02-19T20:46:58Z", "author_association": "MEMBER", "html_url": "https://github.com/mozilla/normandy/pull/40#issuecomment-186398756", "updated_at": "2016-02-19T20:46:58Z", "user": {"following_url": "https://api.github.com/users/mythmon/following{/other_user}", "events_url": "https://api.github.com/users/mythmon/events{/privacy}", "organizations_url": "https://api.github.com/users/mythmon/orgs", "url": "https://api.github.com/users/mythmon", "gists_url": "https://api.github.com/users/mythmon/gists{/gist_id}", "html_url": "https://github.com/mythmon", "subscriptions_url": "https://api.github.com/users/mythmon/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/305049?v=4", "repos_url": "https://api.github.com/users/mythmon/repos", "received_events_url": "https://api.github.com/users/mythmon/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/mythmon/starred{/owner}{/repo}", "site_admin": false, "login": "mythmon", "type": "User", "id": 305049, "followers_url": "https://api.github.com/users/mythmon/followers"}, "body_html": "<p>I'm getting an error on the server: \"The joined path   is located outside of the base path component (/home/mythmon/src/normandy/normandy/media)\"</p>\n<p>The editor isn't showing up now. I am getting 404s for the font awesome css file. I tried running <code>manage.py collectstatic</code>. I'm not sure what's going on.</p>", "id": 186398756, "issue_url": "https://api.github.com/repos/mozilla/normandy/issues/40"}